packages/frontend-shared/src/gql-components/RecordRunModal.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <StandardModal
+    class="transition transition-all duration-200"
+    variant="bare"
+    :title="t('specPage.banners.record.title')"
+    :model-value="isModalOpen"
+    :help-link="helpLink"
+    :no-help="!helpLink"
+    data-cy="record-run-modal"
+    @update:model-value="close"
+  >
+    <div class="max-w-175 py-7 px-6 text-gray-600">
```

Comment:
Add a copy block here that would have the same content as the banner for this implementation.  This allows the `RecordPrompt` to just encapsulate the logic around the TerminalPrompt and the logic to create it for recording a run.